inc/modules/export/namespace.php

```diff
@@ -279,3 +281,47 @@ function update_pins() {
 		wp_send_json_success( $data );
 	}
 }
+
+/**
+ * Get the HTML for "About the Authors" section given a chapter ID.
+ *
+ * @param $chapter_id Integer
+ * @return string
+ */
+function get_contributors_section( $chapter_id ) {
+	$contributors = new Contributors();
+	$chapter_contributors = $contributors->getContributorsWithMeta( $chapter_id, 'authors' );
+	if ( empty( $chapter_contributors ) ) {
+		return '';
+	}
+	$title = sprintf( _n( '%s Author', '%s Authors', count( $chapter_contributors ), 'pressbooks' ), 'About the' );
+	$print = '<div class="contributors">';
+	$print .= "<h3 class=\"about-authors\">{$title}</h3>";
+	foreach ( $chapter_contributors as $contributor ) {
+		$print .= '<div class="contributor_name_and_links">';
+		if ( $contributor['contributor_picture'] ) {
+			$print .= "<img class=\"contributor_profile_picture\" src=\"{$contributor['contributor_picture']}\" />";
```

Comment:
EPUB validation is complaining about missing alt tags here

Reply:
For alt text, we should attempt to include/display the image's alt text and fall back to `alt=""` if none has been set for the image in question. https://webaim.org/techniques/alttext/#decorative